docs/deployment/docker-swarm.md

```diff
@@ -6,8 +6,8 @@ You can create a single-host Docker Swarm on your laptop with a single command.
 
 This is how you initialize your master node:
 
-```
-# docker swarm init
+```bash
+docker swarm init
```

Comment:
All input commands in the docs so far have a dollar prefix.

Reply:
I'm using a markdown linter that uses the following [rule](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md014). I personally like it so that when a person reads the docs they can use the copy/paste feature without getting the extra `$` sign that will need remove to run in a terminal. If you want to keep the `$` I can turn off that lint feature in the future.

Reply:
This is the style I've used throughout all content on OpenFaaS and my blogs:

https://docs.docker.com/network/bridge/#connect-a-container-to-a-user-defined-bridge
https://wiki.archlinux.org/index.php/Installation_guide